recipes/approvaltests.cpp/all/test_package/.gitignore

```diff
@@ -0,0 +1 @@
+*.received.txt
```

Comment:
Is this file needed in this package?

Reply:
`*.received.txt` files are generated as a side effect of running the test. I don't know if that qualifies it as **needed**, but it is a convenience to help prevent accidental inclusion of these test outputs in a commit.

Reply:
Ok, it the automatic checks don't complain... I'm nobody to say anything 😄